chart/values.yaml

```
@@ -51,6 +51,9 @@ base:
   default_req_timeout: 5s
   # -- Cache timeout for core agent & diskpool deployment
   cache_poll_period: 30s
+  logFormat:
+    jsonEnabled: false
+    ansiColorsEnabled: true
```

how about an enum?
And maybe combine with other log stuff such as silence level

```suggestion
  logging:
    format: pretty|compact|json
    color: true
    silenceLevel:
```

---

@tiagolobocastro , Initially thought about it. However, we use `FmtStyle::Compact ` only once while starting a csi node cmd line (freeze/unfreeze) service. Everywhere else its `FmtStyle::Pretty` by default. 

With my changes we keep the same behaviour. If someone wants to have json stdout they enable it or else its whatever style each service defaults to now.

---

Yes but if we want to change it then it becomes a ambiguous:
jsonEnabled: true
compactEnabled: true